pypeit/sensfunc.py

```diff
@@ -420,11 +422,36 @@ def flux_std(self):
         self.sens['SENS_FLUXED_STD_FLAM_IVAR'] = flam_ivar.T
         self.sens['SENS_FLUXED_STD_MASK'] = flam_mask.T
 
+        #save the model that was used
+        model_interp_func = scipy.interpolate.interp1d(self.std_dict['wave'].value, self.std_dict['flux'].value,
+                                                       bounds_error=False, fill_value='extrapolate')
+        model_flux_sav = np.zeros_like(self.sens['SENS_FLUXED_STD_FLAM'])
+        for iorddet in range(self.sens['SENS_FLUXED_STD_WAVE'].shape[0]):
+            wave_gpm = self.sens['SENS_FLUXED_STD_WAVE'][iorddet] > 1.0
+            model_flux_sav[iorddet][wave_gpm] = model_interp_func(self.sens['SENS_FLUXED_STD_WAVE'][iorddet][wave_gpm])
+
+        self.sens['SENS_STD_MODEL_FLAM'] = model_flux_sav
+
     def eval_zeropoint(self, wave, iorddet):
         """
-        Dummy method, overloaded by subclasses
+        Evaluate at a given wavelength the sensitivity function zeropoint for a given order/detector.
+        This is a dummy method, overloaded by subclasses.
+
+        Parameters
+        ----------
+        wave : `numpy.ndarray`_
+            Wavelength array at which to evaluate the zeropoint
+        iorddet : :obj:`int`
+            The order/detector for which to evaluate the zeropoint
+
+        Returns
+        -------
+        zeropoint : `numpy.ndarray`_
+            The zeropoint evaluated given wavelength array and order/detector
+        wave : `numpy.ndarray`_
+            The wavelength array used for the zeropoint evaluation.
         """
-        pass
+        return None
```

Comment:
Given that there are normally two returned object, should this `return None, None`?